src/test/regress/expected/subquery_in_where.out

```diff
@@ -39,17 +39,18 @@ ERROR:  cannot pushdown the subquery
 DETAIL:  Complex subqueries and CTEs are not allowed in the FROM clause when the query has subqueries in the WHERE clause and it references a column from another query
 -- Recurring tuples as empty join tree 
 SELECT * 
-FROM   (SELECT 1 AS id, 
-               2 AS value_1, 
-               3 AS value_3) AS tt1 
+FROM   (SELECT 1 AS id, 2 AS value_1, 3 AS value_3
```

Comment:
Can we leave this change when we work on `RESULT_EMPTY_RTE` (or whatever the name)?

Reply:
pg12 flattens the previous subquery. Some of these tests might be off, but the `RTE_RESULT` change is part of what is allowing pg12 to more aggressively handle these types of queries. It might be correct to create an `_2.out` to keep the single row subquery

Reply:
No, I think it is fine to keep as is for now.

Reply:
By "as is" do you mean with the `UNION` or as it is before this PR?